### PR TITLE
Bump gazel to v10.

### PR DESCRIPTION
**Pull request by spxtr**

### CODE REVIEW

**Comment by ixdy** [comment]
LGTM

File: verify/update-bazel.sh [hunk 1]
```diff
@@ -17,5 +17,5 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-go get -u gopkg.in/mikedanese/gazel.v6/gazel
```
Comment (ixdy) [inline verify/update-bazel.sh]: we might want to vendor here, too.

File: testgrid/config/BUILD [hunk 2]
```diff
@@ -18,9 +18,6 @@ go_binary(
 go_test(
     name = "go_default_test",
     srcs = ["config_test.go"],
-    data = [
-        "config.yaml",
```
Comment (spxtr) [inline testgrid/config/BUILD]: why did it do this?

Comment (ixdy) [inline testgrid/config/BUILD]: that's a good question.

**Comment by spxtr** [comment]
Applying `do-not-merge` until this is solved.

**Comment by spxtr** [comment]
Fixed by not specifying `--root`, which isn't needed anymore anyway.

**Pull request merged**